pkg/api/folder_test.go

```diff
@@ -26,6 +22,9 @@ import (
 	"github.com/grafana/grafana/pkg/services/user"
 	"github.com/grafana/grafana/pkg/setting"
 	"github.com/grafana/grafana/pkg/web/webtest"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
```

Comment:
Why are these being moved?

Reply:
removed the extra spaces in the imports.... then it is the auto sorting 🤷🏻

Reply:
usual layout is core stuff, then 3rd-party libs, then grafana libs, with a blank line between them

Reply:
hymm -- I have just been removing the extra lines and then whatever the IDE says.  If we have any opinion on what it should be ideally that can be a lint rule 😬

Reply:
or leave the blank line alone and the ide will do the right thing ;)

Reply:
Finally updated my settings with:
```
"gopls": {   
        "formatting.local": "github.com/grafana",
        "ui.semanticTokens": true
    },
```
and that seems to organize imports "properly"